humanize-duration.js

```diff
@@ -540,6 +551,19 @@
     }
   }
 
+    // Internal helper function for Slovak language.
+  function getSlovakForm (c) {
```

Comment:
This is identical to the `getCzechForm` helper. Could you delete this one, rename `getCzechForm` to something else (like `getCzechOrSlovakForm`), and use that one everywhere?

Reply:
done!